src/int_set.jl

```diff
@@ -5,12 +5,7 @@ import Base: similar, copy, copy!, eltype, push!, pop!, delete!,
              setdiff, setdiff!, symdiff, symdiff!, in, start, next, done,
              last, length, show, hash, issubset, ==, <=, <, unsafe_getindex,
              unsafe_setindex!, findnextnot, first
-import Compat: popfirst!
-if !isdefined(Base, :complement)
-    export complement, complement!
-else
-    import Base: complement, complement!
-end
+export complement, complement!
```

Comment:
These should move to the main file (`src/DataStructures.j`) with the other exports.
(on their own line like most of the "subpackage" exports)